lib/src/mt_dev.c

```diff
@@ -676,13 +677,15 @@ static struct rte_flow* dev_rx_queue_create_flow(struct mt_interface* inf, uint1
   ipv4_spec.hdr.next_proto_id = IPPROTO_UDP;
 
   if (has_ip_flow) {
-    memset(&ipv4_mask.hdr.dst_addr, 0xFF, MTL_IP_ADDR_LEN);
-    if (mt_is_multicast_ip(flow->dip_addr)) {
-      rte_memcpy(&ipv4_spec.hdr.dst_addr, flow->dip_addr, MTL_IP_ADDR_LEN);
-    } else {
-      rte_memcpy(&ipv4_spec.hdr.src_addr, flow->dip_addr, MTL_IP_ADDR_LEN);
-      rte_memcpy(&ipv4_spec.hdr.dst_addr, flow->sip_addr, MTL_IP_ADDR_LEN);
-      memset(&ipv4_mask.hdr.src_addr, 0xFF, MTL_IP_ADDR_LEN);
+    if (drv_type != MT_DRV_IGC) {
```

Comment:
Seems this is not necessary, `net_igc` already declare it not support `ip flow` in https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/lib/src/mt_dev.c#L64

Reply:
> Seems this is not necessary, `net_igc` already declare it not support `ip flow` in https://github.com/OpenVisualCloud/Media-Transport-Library/blob/main/lib/src/mt_dev.c#L64

Resolve by removing commit.